src/main/java/com/google/sps/data/GameInstance.java

```diff
@@ -9,6 +9,8 @@
   private boolean isActive;
   private String creator;
   private String gameId;
+  private String currentQuestion;
```

Comment:
Nice, we're getting to how to control the state of the game. If we store the `currentQuestion` as a String, this would mean copying from a `Game` to the `GameInstance` right? But we would also need to copy the answers for it to be independent of the `Game`? The other thing I was thinking about is just storing `currentQuestion` as an `int` that just signals which `Question` to retrieve from `Game` in the front end.